src/System.Windows.Forms/src/System/Windows/Forms/ButtonBase.ButtonBaseAccessibleObject.cs

```diff
@@ -26,5 +26,51 @@ public override void DoDefaultAction()
                 owner.OnClick(EventArgs.Empty);
             }
         }
+
+        internal static string? GetKeyboardShortcut(Control control, bool useMnemonic, Label? previousLabel)
+        {
+            char mnemonic = (char)0;
+
+            if (!useMnemonic && previousLabel is not null && previousLabel.UseMnemonic)
+            {
+                string text = previousLabel.Text;
+                if (!string.IsNullOrEmpty(text))
+                {
+                    mnemonic = WindowsFormsUtils.GetMnemonic(text, false);
+                }
```

Comment:
```suggestion
            mnemonic = WindowsFormsUtils.GetMnemonic(previousLabel.Text, false);
```

You do not need explicit check. `GetMnemonic` handle null or empty strings.